app/src/main/res/values/strings.xml

```diff
@@ -186,6 +187,7 @@
     <string name="pref_summary_colored_navigation_bar">Colors the navigation bar in the primary color.</string>
     <string name="pref_summary_colored_app_shortcuts">Colors the app shortcuts in the primary color.</string>
     <string name="pref_summary_audio_ducking">Notifications, navigation etc.</string>
+    <string name="pref_summary_synchronized_lyrics_show">Show synchronized lyrics for song if available</string>
```

Comment:
prefer the same name as the music file in the same folder, maybe be easy to implementation.
what's the lrc file charset? how about forcing the UTF-8 (can handle all strings), only for easy implementation.